crow/src/distributions/distributionNDCartesianSpline.cxx

```diff
@@ -156,14 +156,16 @@
       if (_cdf_provided){
         throwError("BasicMultiDimensionalCartesianSpline Distribution error: inverseMarginal calculation not available if CDF provided");
       }else{
-        value = _interpolator.splineCartesianInverseMarginal(f, dimension, 0.01);
+        value = _interpolator.splineCartesianInverseMarginal(f, dimension, 1.e-6);
```

Comment:
Should we remove the coded value, and add a variable to control that?

Reply:
yes this is the final plan